pkg/sideinput/examples/simple-sideinput/pipeline.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: numaflow.numaproj.io/v1alpha1
+kind: Pipeline
+metadata:
+  name: my-pipeline
+spec:
+  sideInputs:
+    - name: myticker
+      container:
+        image: "quay.io/numaio/numaflow-go/sideinput:v0.5.0"
+        imagePullPolicy: Always
+      trigger:
+        schedule: "*/2 * * * *"
+        # interval: 180s
+        # timezone: America/Los_Angeles
+  vertices:
+    - name: in
+      source:
+        # A self data generating source
+        generator:
+          rpu: 1
+          duration: 1s
+    - name: si-log
+      udf:
+        container:
+          image: "quay.io/numaio/numaflow-go/udf-sideinput:v0.5.0"
+          imagePullPolicy: Always
+          env:
+            - name: SIDEINPUT_PATH
+              value: "/var/numaflow/side-inputs"
+            - name: SIDEINPUT_NAME
+              value: "myticker"
```

Comment:
this also.. default name will be the same as
```
      sideInputs:
        - myticker
 ```

correct?... maybe add some comments?...